politeiad/backend/gitbe/gitbe.go

```diff
@@ -2122,6 +2122,23 @@ func (g *gitBackEnd) Inventory(vettedCount, branchCount uint, includeFiles bool)
 	return pr, br, nil
 }
 
+// GetPlugins returns the decred plugin settings.
```

Comment:
This doc should probably be more generic, like "GetPlugins returns a list of currently supported plugins and their settings."